src/components/layout/ZooHeader.jsx

```diff
@@ -23,8 +24,14 @@ const ZooHeader = (props) => {
 
 ZooHeader.defaultProps = {
   authContainer: null,
-  logoHomeLink: <ZooniverseLogo height="1.25em" width="1.25em" />,
-  mainHeaderNavList: null
+  logoHomeLink: <Anchor className="site-header__link" href="http://www.zooniverse.org"><ZooniverseLogo height="1.25em" width="1.25em" /></Anchor>,
+  mainHeaderNavList: [
+    <Anchor className="site-header__link--small" href="http://www.zooniverse.org/projects" label="Projects" />,
+    <Anchor className="site-header__link--small" href="http://www.zooniverse.org/about" label="About" />,
+    <Anchor className="site-header__link--small" href="http://www.zooniverse.org/about" label="About" />,
```

Comment:
We have two About links?

Reply:
One of those should've been the link Get Involved. Oops!